core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -194,7 +194,8 @@ class NetworkAnalyzer {
 
       // Assume everything before sendStart was just DNS + (SSL)? + TCP handshake
       // 1 RT for DNS, 1 RT (maybe) for SSL, 1 RT for TCP
-      let roundTrips = 2;
+      let roundTrips = 1;
+      if (!record.protocol.startsWith('h3')) roundTrips += 1; // TCP
```

Comment:
We should add this logic to [TTFB](https://github.com/GoogleChrome/lighthouse/blob/main/core/computed/metrics/time-to-first-byte.js#L28) as well.

WDYT about adding some kind of `getRoundTripCount` helper function to `NetworkRequest` so that we don't have to maintain this in 3 different places?

Reply:
It's a bit more complicated than first glance to share this code, since the analyzer has its own concept of "is this connection reused?". I can try in a followup.